static/js/index.js

```diff
@@ -0,0 +1,42 @@
+const flaskapi_url = 'http://127.0.0.1:5000/fetchEmployee/'
+
+async function loadData(url) {
+    const response = await fetch(url)
+    let data = await response.json();
+    return data;
+}
+
+async function fetchEmployee(id){
+    const url_emp = flaskapi_url + id
+    data = await loadData(url_emp)
+    console.log("data from the backend is => ", data);
+    document.getElementById("firstName").value = data.firstName;
+    document.getElementById("lastName").value = data.lastName;
+    document.getElementById("DOB").value = data.dob;
+    document.getElementById("hireDate").value = data.hireDate;
+    document.getElementById("department").value = data.department;
+    document.getElementById("salary").value = data.salary;
+    document.getElementById("manager").value = data.manager;
+    document.getElementById("designation").value = data.designation;
+
+}
+
+function handleSearch(){
+    const empId = document.getElementById('empId').value;
+    console.log("empID = ", typeof(empId));
+    if(empId < 1 || empId > 3 || empId == ''){
```

Comment:
The user can type in anything in the search tab so the developer needs to provide more validations before a search query is fired. The appropriate validation to use here should be that empId should be a number only. I can also search for text which should ideally show an alert: "Incorrect Employee Id Entered!"